src/main/kotlin/org/ptss/support/domain/models/Questionnaire.kt

```diff
@@ -0,0 +1,26 @@
+package org.ptss.support.domain.models
+
+class Questionnaire() {
+    var id: String = ""
+    var title: String = ""
+    var description: String = ""
+    var estimatedTimeOfCompletion: String = ""
+    var assignedAt: String = ""
+    var isFinished: Boolean = false
```

Comment:
does this need to be a Boolean, or a boolean?
Difference between Boolean (with uppercase B) it can be set to null.
boolean can only be true or false. Default value being false. So if it should never be null, i'd suggest making it a boolean and removing the "= false"

Reply:
didn't know that but it doesn't let me change it to lowercase? GH copilot also doesnt bring up the lowercase
![image](https://github.com/user-attachments/assets/8bdf3c4b-567a-4239-89bf-29ba0de9b0f5)

Reply:
`Boolean` is niet nullable, is wel de case in Java vlgns mij dus daarom de verwarring van Tayam. `Boolean?` is een nullable boolean type, dus niks aan t handje.

Reply:
allemaal weer wat geleerd I guess, dan is dit opgelost